docker/test/common-services.yml

```diff
@@ -0,0 +1,24 @@
+version: '2.1'
+
+services:
+  ganache:
+    image: trufflesuite/ganache-cli
+    command: ganache-cli --gasLimit 90000000000 --gasPrice 1 -a 10 -m "vanish junk genuine web seminar cook absurd royal ability series taste method identify elevator liquid"
+    ports:
+      - 8545:8545
+
+  signer-node:
```

Comment:
It would make a lot of sense to move the command here as well.

Reply:
opened #1418